config/locales/image.es.yml

```diff
@@ -0,0 +1,35 @@
+en:
+  hyrax:
+    icons:
+      image:     'fa fa-picture-o'
+    select_type:
+      image:
+        name:               "Imagen"
+        description:        "La imagen de una o varias páginas funciona."
+  simple_form:
+    labels:
+      image:
```

Comment:
This can be removed as they should be the same as the provided defaults: https://github.com/projecthydra-labs/hyrax/blob/master/config/locales/hyrax.es.yml#L647